users/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+from users.models import User
+
+class RegistrationSerializer(serializers.ModelSerializer):
+
+     class Meta:
+        model = User
+        fields = ['email','name','institution','country_code','phone']
+         
+         
+        def save(self):
+            user = User(
```

Comment:
Please take password field also from the user. Also check the password length of minimum 8 chars.